docker-compose.yml

```diff
@@ -10,13 +10,19 @@ services:
     tty: true
     volumes:
       - ./.env:/home/node/app/.env
+    environment:
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+
```

Comment:
I wonder if this is actually doing something, probably those values are also the defaults, since my understanding is that `=` will not work in this context it should be `:`, could you verify?

Reply:
changed, please check